src/components/expression/expression-panel.html

```diff
@@ -30,6 +30,10 @@
       <button id="removeCondition" class="btn btn-default btn-xs glyphicon glyphicon-minus" title="remove conditions"></button>
     </div>
   </div>
+  <div id="zoom-out" class="panel-section">
+    <div class="item-title">Zoom Out</div>
+    <button id="out" class="zoom-out btn btn-default btn-xs glyphicon glyphicon-minus"></button>
```

Comment:
Probably too many ids than necessary. Ids are useful unique identifiers but don't use too many of them. In this case you can just keep zoom-out and then use #zoom-out > button.